docs/content/preview/reference/configuration/insecure-yugabyted-examples.md

```diff
@@ -0,0 +1,939 @@
+
+```sh
+./bin/yugabyted start --advertise_address=<host-ip> --join=<ip-address-first-yugabyted-node> --cloud_location=aws.us-west.us-west-1a --fault_tolerance=region
+```
+```sh
+./bin/yugabyted start --advertise_address=<host-ip> --join=<ip-address-first-yugabyted-node> --cloud_location=aws.us-central.us-central-1a --fault_tolerance=region
+```
+
+After starting the yugabyted processes on all the nodes, configure the data placement constraint of the YugabyteDB cluster as follows:
+
+```sh
+./bin/yugabyted configure data_placement --fault_tolerance=region
+```
+
+This command determines the data placement constraint based on the `--cloud_location` of each node in the cluster. If there are 3 or more regions available in the cluster, the command configures the cluster to survive at least 1 region failure. Otherwise, it outputs a warning message.
+
+The replication factor of the cluster defaults to 3.
+
+You can set the data placement constraint manually using the `--constraint_value` flag, which takes the comma-separated value of `cloud.region.zone` as follows:
+
+```sh
+./bin/yugabyted configure data_placement --fault_tolerance=region --constraint_value=aws.us-east.us-east-1a,aws.us-west.us-west-1a,aws.us-central.us-central-1a
+```
+
+You can set the replication factor of the cluster manually using the `--rf` flag as follows:
+
+```sh
+./bin/yugabyted configure data_placement --fault_tolerance=region --constraint_value=aws.us-east.us-east-1a,aws.us-west.us-west-1a,aws.us-central.us-central-1a --rf=3
+```
+
+### Enabling and Disabling encryption at rest
+
+Create a cluster using the desired deployment example.
+
+To enable encryption at rest in a deployed local cluster, run the following command:
+```sh
+./bin/yugabyted configure encrypt_at_rest --enable --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node1
+```
+
+To enable encryption at rest in a deployed multi-zone or multi-region cluster, run the following command from any VM:
+```sh
+./bin/yugabyted configure encrypt_at_rest --enable
+```
+
+To disable encryption at rest in a local cluster with encryption at rest enabled, run the following command:
+```sh
+./bin/yugabyted configure encrypt_at_rest --disable --base_dir=$HOME/yugabyte-{{< yb-version version="preview" >}}/node1
+```
+
+To disbale encryption-at-rest in a multi-zone or multi-region cluster with encryption-at-rest enabled, run the following command from any VM:
+```sh
+./bin/yugabyted configure encrypt_at_rest --disable
+```
+
+### Pass additional flags to YB-TServer
+
+Create a single-node cluster and set additional flags for the YB-TServer process.
+
+```sh
+./bin/yugabyted start --tserver_flags="pg_yb_session_timeout_ms=1200000,ysql_max_connections=400"
+```
+
+## Upgrade a YugabyteDB cluster
+
+To use the latest features of the database and apply the latest security fixes, it's prudent to upgrade your YugabyteDB cluster to the [latest release](https://download.yugabyte.com/).
```

Comment:
To use the latest YugabyteDB features and apply the latest security updates, it is recommended to upgrade your YugabyteDB cluster to the [latest release](https://download.yugabyte.com/).